packages/terra-form-select/src/shared/_FrameUtil.js

```diff
@@ -51,6 +54,7 @@ class FrameUtil {
       isAbove,
       isPositioned: true,
       top: semanticTopWhenAbove,
+      resizeOnSearch: false,
```

Comment:
I think we can simplify the logic by applying a `bottom` instead of a `top` when the dropdown is appearing above and isTouchAccessible is enabled. 

```jsx
// Extract the height of the target.
const { height: targetHeight } = target.getBoundingClientRect();

// If isTouchAccessible is enabled and the dropdown should be above, apply a bottom with the same value as the target height.
const semanticBottomWhenAbove = (useSemanticDropdown && isAbove ? targetHeight : undefined);
```

The bottom value does not need to be recalculated when the height of the dropdown changes because it's bound to the bottom regardless of height it stays attached to the target. This also removes additional resize logic.

Reply:
I had actually tried it initially but without top being set the dropdown did not open above the select field.

Reply:
It works, there are a few other tweaks that need to be done. Likely the issue you were encountering was from this still being set:

https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/shared/_Dropdown.module.scss#L27

Reply:
Done a2c35f5